main.go

```diff
@@ -303,23 +305,28 @@ func main() {
 	//
 	// Create client
 	fmt.Println()
-	log.Infof("Authenticateing")
+	log.Infof("Authenticating")
 
 	jwtConfig := new(jwt.Config)
 
 	if configs.JSONKeyPath != "" {
-		jsonKeyPth := ""
+		jsonKeyPth := configs.JSONKeyPath
 
-		if strings.HasPrefix(configs.JSONKeyPath, "file://") {
+		url, err := url.Parse(configs.JSONKeyPath)
+		if err != nil {
+			failf("Failed to parse json key path, error: %s", err)
```

Comment:
Please don't do this like that. Handle if the path is just a relative path, for any case, not just for p12 or something like that!